### PR TITLE
CONSOLE-4124: replace deprecated components in SilenceForm

### PR DESCRIPTION
**Pull request by rhamilto**
**Note:**  [CONSOLE-4124](https://issues.redhat.com//browse/CONSOLE-4124) will be addressed by 4 PRs, of which this is 1.

After

https://github.com/user-attachments/assets/2031bd11-cddb-4256-8078-4fb7f9ed3a34

### CODE REVIEW

**Comment by kyoto** [comment]
/lgtm

FYI @jgbernalp @zhuje @PeterYurkovich 

A similar change will be needed in monitoring plugin (https://github.com/openshift/monitoring-plugin/blob/main/web/src/components/silence-form.tsx)

File: frontend/public/components/monitoring/silence-form.tsx [hunk 1]
```diff
@@ -280,18 +287,28 @@ const SilenceForm_: React.FC<SilenceFormProps> = ({ defaults, Info, title }) =>
...
+                    >
                       {duration}
-                    </DropdownToggleDeprecated>
-                  }
-                />
+                    </MenuToggle>
+                  )}
+                  onOpenChange={setIsOpen}
+                >
+                  {selectOptions}
```
Comment (rhamilto) [inline frontend/public/components/monitoring/silence-form.tsx:310]: Missing `<SelectList>`

**Pull request merged**